test/core/number/cos.wren

```diff
@@ -0,0 +1,6 @@
+IO.print(0.cos)             // expect: 1
+IO.print(Num.pi.cos)        // expect: -1
+IO.print((2*Num.pi).cos)    // expect: 1
```

Comment:
Of course I realize doubles have limited precision. With "precise results" I meant that we should be able to expect exactly the same outcome each time we run the application. Maybe I should have called it "consistent results" instead. If the results are consistently the same imprecise numbers, then there is no need to resort to fuzzy comparison in a test.

So what I was basically asking, is whether we can expect these functions to return different results on different platforms, or whether the standard is strict enough to ensure that it's the same imprecise result everywhere.